src/controller/SDLController.cpp

```diff
@@ -57,39 +57,12 @@ bool SDLController::Close() {
 }
 
 void SDLController::NormalizeStickAxis(SDL_GameControllerAxis axisX, SDL_GameControllerAxis axisY,
-                                       int16_t axisThreshold, int32_t virtualSlot) {
-    auto profile = getProfile(virtualSlot);
-
+                                       int32_t virtualSlot) {
     const auto axisValueX = SDL_GameControllerGetAxis(mController, axisX);
     const auto axisValueY = SDL_GameControllerGetAxis(mController, axisY);
 
-    // scale {-32768 ... +32767} to {-84 ... +84}
-    auto ax = axisValueX * 85.0f / 32767.0f;
-    auto ay = axisValueY * 85.0f / 32767.0f;
-
-    // create scaled circular dead-zone in range {-15 ... +15}
-    auto len = sqrt(ax * ax + ay * ay);
-    if (len < axisThreshold) {
-        len = 0.0f;
-    } else if (len > 85.0) {
-        len = 85.0f / len;
-    } else {
-        len = (len - axisThreshold) * 85.0f / (85.0f - axisThreshold) / len;
-    }
-    ax *= len;
-    ay *= len;
-
-    // bound diagonals to an octagonal range {-68 ... +68}
-    if (ax != 0.0f && ay != 0.0f) {
-        auto slope = ay / ax;
-        auto edgex = copysign(85.0f / (abs(slope) + 16.0f / 69.0f), ax);
-        auto edgey = copysign(std::min(abs(edgex * slope), 85.0f / (1.0f / abs(slope) + 16.0f / 69.0f)), ay);
-        edgex = edgey / slope;
-
-        auto scale = sqrt(edgex * edgex + edgey * edgey) / 85.0f;
-        ax *= scale;
-        ay *= scale;
-    }
+    auto ax = axisValueX * MAX_AXIS_RANGE / 32767.0f;
```

Comment:
Should probably keep the comment
```c
// scale {-32768 ... +32767} to {-84 ... +84}
```

Reply:
i think it'd be nice to just #define `32767.0f` as `MAX_SDL_RANGE` or something
```c
#define MAX_SDL_RANGE (float)INT16_MAX
```

Reply:
I think it makes sense to add it in the SDLController header. A similar value as well in the Wii U header.